_i18n/de/resources/moneropedia/bootstrap-node.md

```diff
@@ -1,13 +1,13 @@
 ---
-terms: ["bootstrap-node", "bootstrap-nodes"]
-summary: "A node to which a daemon connects to give immediate usability to wallets while syncing"
+terms: ["bootstrap-node", "bootstrap-nodes", "Bootstrap-Node", "Bootstrap-Nodes"]
+summary: "Ein Node, zu dem ein Hintergrunddienst verbindet, um schon während des Synchronisierens eine sofortige Verwendbarkeit von Wallets zu ermöglichen"
```

Comment:
Statt Node könnte man auch Knoten sagen, falls Node übersetzt werden sollte? Ist gebräuchlich - siehe z.B. "Netzwerkknoten", oder in der Graphentheorie werden "Nodes" auch als "Knoten" bezeichnet.

Reply:
You're right, it would be possible to translate "node" to "(Netzwerk-)Knoten", but iirc after a discussion it was decided to stick to "Node" (which is also listed in the German terminology guide) after "Knoten" was used by some translators and the result were inconsistent translations. Feel free to open a new issue to suggest an "official" translation which can be updated in the terminology guide.